ui/guest/src/util.js

```diff
@@ -523,3 +526,13 @@ export function removeLastPiece(str, splitChar = '.') {
 export function popPiece(str, splitChar = '.') {
   return str.substr(str.lastIndexOf(splitChar) + 1);
 }
+
+function decodeHTML(html) {
+  var txt = document.createElement('textarea');
+  txt.innerHTML = html;
+  return txt.value;
+}
+
+export function stripEscapedHtmlTags(str) {
+  return decodeHTML(str).replace(/<[^>]*>/g, '');
+}
```

Comment:
Do we still need this? Provided that the service should give you clean ready-to-use data.

Reply:
yes, will talk about it